CHANGELOG.md

```diff
@@ -30,6 +30,9 @@
 * Admin bar menu items can now specify a custom Vue component to be used in place of `AposButton`.
 * Sets `username` fields to follow the user `title` field to remove an extra step in user creation.
 * Adds default data to the `outerLayoutBase.html` `<title>` tag: `data.piece.title or data.page.title`.
+* The new `._id(_idOrArrayOfIds)` query builder replaces `explicitOrder` and accepts an array of document `_id`s or a single one. `_id` can be used as a multivalued query parameter. Documents are returned in the order you specify, and just like with single-document REST GET requests, the locale of the `_id`s is overridden by the `apos-mode` query parameter if present.
+* The `.published(true)` query builder adds a `_publishedDoc` property to each returned draft document that has a published equivalent. `published=1` can be used as a query parameter. Note this is not the way to fetch only published documents. For that, use `.locale('en:published')` or similar.
```

Comment:
The naming of this is confusing. It's more like `includePublished` or better, `withPublished`. Something like that. `addPublished`? As-is it seems like it'd return the published doc.

Reply:
This note needs to be updated.

Reply:
This changelog entry is still outdated. `.published` to `.withPublished`